TestON/drivers/common/cli/onosclidriver.py

```diff
@@ -1545,6 +1545,27 @@ def getAllIntentsId( self ):
             main.cleanup()
             main.exit()
 
+
+    def FlowStateCount( self, id ):
```

Comment:
Can I suggest renaming this to something addedFlowCount or flowAddedCount? The name makes we think it would return the count of the flows in each state. Which is something I eventually plan to put in my HA Tests